### PR TITLE
[27] Add component for Weight

### PR DESCRIPTION
**Pull request by bobbylej**
- Create component for Weight
- Use Weight component in converter for markdown
- Pass color props in converter for markdown
- Modify props and styles in Circle
- Modify styles in AsideGraphicEditor

### CODE REVIEW

File: src/app/shared/components/Weight/Weight.styles.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { Theme } from '@material-ui/core';
+import { makeStyles } from '@material-ui/styles';
+import { pxToRem } from 'app/shared/utils/stylesUtils';
+
+export const useWeightStyles = makeStyles((theme: Theme) => ({
+  root: {
+    display: 'inline-block',
+    height: pxToRem(27),
+    border: `2px solid ${theme.palette.primary.main}`,
+    borderRadius: '999rem',
```
Comment (lukaszzadamczyk) [inline src/app/shared/components/Weight/Weight.styles.ts:10]: 999rem ? 😁

File: src/app/shared/components/Weight/Weight.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    rootSecondary,
+    rootSecondaryLight,
+  } = useWeightStyles();
+  const [value, setValue] = useState(weight);
+
+  const onChangeValue = (
+    event: ChangeEvent<HTMLTextAreaElement | HTMLInputElement>,
+  ): void => {
+    const newValue = +event.target.value;
+    setValue(newValue);
```
Comment (lukaszzadamczyk) [inline src/app/shared/components/Weight/Weight.tsx:30]: what does this + mean? 😁

**Review by lukaszzadamczyk — APPROVED** [review]
You changed my components a bit 😂, but overall it looks nice😊

File: src/app/shared/components/Weight/Weight.styles.ts (hunk 1 above)
Comment (bobbylej) [reply src/app/shared/components/Weight/Weight.styles.ts:10]: yes, it makes border radius round but not elipsis 😉 
`border-radius: 999rem` looks like that:
Zrzut ekranu 2021-05-17 o 21 56 28
and `border-radius: 50%` looks like that:
Zrzut ekranu 2021-05-17 o 21 56 46

File: src/app/shared/components/Weight/Weight.tsx (hunk 2 above)
Comment (bobbylej) [reply src/app/shared/components/Weight/Weight.tsx:30]: it converts string to number 😉

**Pull request merged**